lesson03/index.js

```diff
@@ -0,0 +1,17 @@
+const imageUrl = ['bookmark.png', 'message.png'];
+const ul = document.getElementById('js-list');
+const fragment = document.createDocumentFragment();
+
+for (let i = 0; i < imageUrl.length; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.href = `a${i+1}.html`;
+    a.textContent = `a${i+1}`;
+    img.src = `/img/${imageUrl[i]}`;
```

Comment:
nits
Rather than adding 1, I thought it would be better to set the default value to 1.
But I'll leave it up to you to fix it.
```suggestion
for (let i = 1; i < imageUrl.length; i++) {
    const li = document.createElement('li');
    const a = document.createElement('a');
    const img = document.createElement('img');

    a.href = `a${i}.html`;
    a.textContent = `a${i}`;
    img.src = `/img/${imageUrl[i]}`;
```